test/integration/ClientShutdownTest.js

```diff
@@ -36,9 +36,9 @@ describe('ClientShutdownTest', function () {
 
     afterEach(async function () {
         sandbox.restore();
-        if (cluster !== null) {
+        if (cluster !== undefined) {
```

Comment:
I changed these values from null to undefined to fix test fails due to equality operator change([eqeqeq](https://eslint.org/docs/rules/eqeqeq) rule)